examples/chef/devices/rootnode_doorlock_aNKYAreMXE.matter

```diff
@@ -2756,7 +2756,7 @@ endpoint 1 {
     ram      attribute autoRelockTime default = 0;
     ram      attribute soundVolume default = 0;
     ram      attribute operatingMode default = 0;
-    ram      attribute supportedOperatingModes default = 0xFFFF;
+    ram      attribute supportedOperatingModes default = 0x16;
```

Comment:
Could you explain valid value here?

0xFFFF seems to be "nothing supported" according to the spec:

```
For the OperatingModesBitmap, a bit SET indicates that the operating mode IS NOT supported. A bit CLEAR indicates that the operating mode IS supported. This is the inverse of most bitmaps in this specification, and it is RECOMMENDED that clients carefully take this into consideration.
```

so this new default seems to say "i will support this future flag that I do not know about" which seems wrong.